addons/account/models/account_move.py

```diff
@@ -3651,6 +3670,14 @@ def _check_accountable_required_fields(self):
             elif not line.account_id:
                 raise ValidationError("Missing required account on accountable invoice line.")
 
+    @api.constrains('debit', 'credit', 'amount_currency', 'currency_id')
+    def _check_debit_credit_sign(self):
+        for line in self:
+            if line.move_id.use_storno_accounting and (line.debit or line.credit) > 0.0:
+                raise ValidationError(_("Debit / Credit must be negative when dealing with Storno accounting."))
```

Comment:
Wtf? It can be, but it doesn't have to be. Or did I miss something?

Reply:
I would call this `is_storno_inverted` or something like that. Because all the moves of the company use storno accounting.

Reply:
@william-andre Not all the moves will be with storno accounting, only the credit notes from invoices will be on the same side with "minus", also for stock accounting, the return will also generate move lines with "minus" on the same side.

The rest of the operations, normal ones (no returns or credit notes) are acting the same like in contra accounting.

Reply:
yep :facepalm: